### ISSUE TITLE
VS Code Testing UI Bug, test fail output "═════..."

### ISSUE THREAD

**Issue by lewinpauli** [Dart-Code/Dart-Code#5353]
**Describe the bug**
When writing a test that is not inside /test folder, the output of a failing test is "═════..." (take a look at the screenshot)

the debug console displays the correct exception

**To Reproduce**
flutter create

enter the sample code into the test file integration_test/widget_test.dart

hint: the official flutter tutorial for integration tests also creates it inside /integration_test folder
https://docs.flutter.dev/testing/integration-tests#create-the-integration-test-files 

and another hint, because this is a flutter vscode bug I was advised to open the issue here
https://github.com/flutter/flutter/issues/159264

**Expected behavior**
Displaying the correct exception into the vscode ui

**Code Sample**

```dart
import 'package:flutter/material.dart';
import 'package:flutter_test/flutter_test.dart';

Future<void> main() async {
  group(
    "integration test",
    () {
      testWidgets("test1", (WidgetTester tester) async {
        await tester.tap(find.byKey(Key("NotExistingKey")));
      });
    },
  );
}
```

**Screenshots**
![image](https://github.com/user-attachments/assets/7b87fb51-4a5c-4fe4-b843-fad1e52738db)

**Diagnostic Data**
 - Operating System and version: Linux Fedora 41 Gnome
 - VS Code version:  linux 1.95.3
 - Dart extension version: 3.100.0
 - Dart SDK version:  3.5.4 
 - Flutter SDK version: 3.24.5

**Comment by DanTup** [comment]
Thanks, I can reproduce this. It looks like `flutter test` gives different output for integration tests... We usually show the last print or error message in this popup (excluding any that just say "Test failed, see exception logs above). For the unit tests, the only other print message is the exception text, but in the integration tests we get lots of print messages, ultimately ending with this unhelpful line.

![image](https://github.com/user-attachments/assets/32c26207-342d-454f-b5c3-01693df2f738)

I'll try to track down why these are different and if we can't change it, we'll have to find a different way to decide what to show here.

**Comment by DanTup** [comment]
This is now fixed:

![image](https://github.com/user-attachments/assets/9aff6cf6-fc80-41a2-b918-197ea910787a)

If you want to try it out before the next stable release, you can switch to the pre-release extension released just now (v3.101.20241127) in VS Code:

<img width="703" alt="Dart pre-release versions" src="https://user-images.githubusercontent.com/1078012/218796750-90a70b0b-4957-468c-8eb4-688095cde112.png">

**Comment by lewinpauli** [comment]
@DanTup thank you for this very fast fix :)
